.github/workflows/builds.yml

```diff
@@ -16,51 +16,35 @@ jobs:
     name: Trigger snap build
     runs-on: ubuntu-24.04
     if: ${{ github.repository == 'canonical/lxd-pkg-snap' && github.actor != 'dependabot[bot]' }}
+    env:
+      SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+      PACKAGE: "lxd"
+      REPO: "git+ssh://lxdbot@git.launchpad.net/~canonical-lxd/lxd"
+      BRANCH: >-
+        ${{ fromJson('{
+          "main": "latest-edge",
```

Comment:
there's no main branch in this repo, you can just use the branch name

Reply:
Thanks, fixed